contracts/Core/StakeManager.sol

```diff
@@ -74,19 +76,27 @@ contract StakeManager is Initializable, ACL, StakeStorage {
         _;
     }
 
-    /// @param schAddress The address of the Schelling token ERC20 contract
+
+    /// @param razorAddress The address of the Razor token ERC20 contract
     /// @param rewardManagerAddress The address of the RewardManager contract
+
     /// @param voteManagersAddress The address of the VoteManager contract
     /// @param parametersAddress The address of the StateManager contract
     function initialize (
-        address schAddress,
+
+        address razorAddress,
         address rewardManagerAddress,
+
         address voteManagersAddress,
         address parametersAddress
     ) external initializer onlyRole(DEFAULT_ADMIN_ROLE)
     {
-        sch = SchellingCoin(schAddress);
+
+        razor = RAZOR(razorAddress);
         rewardManager = IRewardManager(rewardManagerAddress);
+
```

Comment:
Hey @adi44 
Throughout the PR, a lot of empty lines are inserted.
Please remove them.

Reply:
I still see empty lines here